z2pack/fp/_first_principles.py

```diff
@@ -55,13 +69,17 @@ def __init__(
         command,
         executable=None,
         build_folder='build',
+        scf_folder='scf',
         file_names=None,
         mmn_path='wannier90.mmn',
+        dmn_path='wannier90.dmn',
+        xml_path='pwscf.xml',
```

Comment:
The ``scf_folder`` and ``xml_path`` options seem problematic to me, since they only apply (in this form) for Quantum Espresso calculations.